<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elwin</class>
 <widget class="QWidget" name="Elwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layoutElwin">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="MantidQt::MantidWidgets::MWRunFiles" name="dsInputFiles" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>41</verstretch>
         </sizepolicy>
        </property>
        <property name="findRunFiles" stdset="0">
         <bool>false</bool>
        </property>
        <property name="label" stdset="0">
         <string>Input File</string>
        </property>
        <property name="multipleFiles" stdset="0">
         <bool>true</bool>
        </property>
        <property name="fileExtensions" stdset="0">
         <stringlist>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loElwinOptions">
     <item>
      <layout class="QVBoxLayout" name="properties"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="loPlotAndOptions">
       <item>
        <layout class="QHBoxLayout" name="loPreviewSelection">
         <item>
          <widget class="QLabel" name="lbPreviewFile">
           <property name="text">
            <string>Preview file:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbPreviewFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbPreviewSpec">
           <property name="text">
            <string>Spectrum:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spPreviewSpec">
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlot" native="true">
         <property name="canvasColour" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loSampleLog">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SE log name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLogName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="leLogValue">
       <item>
        <property name="text">
         <string>last_value</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>average</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
